src/main/java/core/basesyntax/FiguresExpo.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class FiguresExpo {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[10];
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        for (int i = 0; i < figures.length; i++) {
+            if (i < 5) {
```

Comment:
what if we would have only 2 figures? think about how to make this code more universal